src/clj/game/cards/ice.clj

```diff
@@ -272,6 +272,16 @@
    "Asteroid Belt"
    (space-ice end-the-run)
 
+   "Authenticator"
+   {:implementation "Encounter effect is manual"
+    :abilities [give-tag]
+    :runner-abilities [{:label "Take 1 tag"
+                        :delayed-completion true
+                        :effect (req (system-msg state :runner "takes 1 tag on encountering Authenticator to Bypass it")
+                                     (tag-runner state :runner eid 1))}]
```

Comment:
Adding `{:unpreventable true}` to this tag is advised--Runner can't bypass if they don't actually take the tag.